### PR TITLE
Katacoda shutdown notice for the Kubernetes blog

### PR DESCRIPTION
**Pull request by natalisucks**
SIG Docs has confirmed with O'Reilly that Katacoda tutorials for the Kubernetes community will be shutting down on 31 March, 2023. This blog post serves as an announcement to the community, 6 weeks before the intended shutdown date.

[Preview](https://deploy-preview-39434--kubernetes-io-main-staging.netlify.app/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/)

Related to #33936 and #38785.

### CODE REVIEW

**Comment by natalisucks** [comment]
This PR replaces #39414 due to my mistake of opening a PR upstream. This is now from my own fork – apologies!

/assign @sftim

**Review by sftim — COMMENTED** [review]
The filename `content/en/blog/_posts/Free-Katacoda-Kubernetes-Tutorials-Are-Shutting-Down` is also wrong.

it should be:
`content/en/blog/_posts/free-katacoda-kubernetes-tutorials-are-shutting-down.md`

**Comment by reylejano** [comment]
I redeployed the build.
nit: should we include the GH discussion https://github.com/kubernetes/website/discussions/38878

**Comment by sftim** [comment]
I'd like to mention the discussion. I think we could merge even without it.

File: content/en/blog/_posts/free-katacoda-kubernetes-tutorials-are-shutting-down.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+[Katacoda](https://katacoda.com/kubernetes), the popular learning platform from O’Reilly that has been helping people learn all about 
+Java, Docker, Kubernetes, Python, Go, C++, and more, [shut down for public use in June 2022](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html). 
+However, tutorials specifically for Kubernetes, linked from the Kubernetes website for our project’s 
+users and contributors, remained available and active after this change. Unfortunately, this will no 
+longer be the case, and Katacoda tutorials for learning Kubernetes will cease working after March 31st, 2023.
+
+The Kubernetes Project wishes to thank O'Reilly Media for the many years it has supported the community 
+via the Katacoda learning platform. You can read more about [the decision to shutter katacoda.com][https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html) 
+on O'Reilly's own site. With this change, we’ll be focusing on the work needed to remove links to 
+their various tutorials. We have a general issue tracking this topic at [#33936](https://github.com/kubernetes/website/issues/33936). We’re also
```
Comment (reylejano) [inline content/en/blog/_posts/free-katacoda-kubernetes-tutorials-are-shutting-down.md:20]: non-blocking suggestion:
```suggestion
their various tutorials. We have a general issue tracking this topic at [#33936](https://github.com/kubernetes/website/issues/33936) and [GitHub discussion](https://github.com/kubernetes/website/discussions/38878). We’re also 
```

**Comment by reylejano** [comment]
/retest
@natalisucks , you can push an empty commit with `git commit --allow-empty -m "empty commit"` then push to your fork to re-trigger the checks
/label tide/merge-method-squash
/lgtm
Remove the hold when ready

**Comment by reylejano** [comment]
Re-applying lgtm
/lgtm

**Comment by natalisucks** [comment]
Apologies, I did one final fix as I noticed on the preview that a link was busted via a missing bracket. Apologies for the reapplying of LGTMs! Appreciate the reviews @reylejano and @sftim 🙏

**Comment by reylejano** [comment]
Checked all the links and they all work, thank you @natalisucks 
Unholding the PR, further updates are welcome as follow-up PR(s)
/unhold

**Pull request merged**